docs/data/indexers/build-your-own/token-transfer-processor/README.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Token Transfer Processor
+sidebar_position: 0
+---
+
+## Overview
+
+The Token Transfer Processor (TTP) is a [package](https://github.com/stellar/go/tree/ttp-v1.0.0/ingest/processors/token_transfer) which uses the [ingest-sdk](../ingest-sdk/README.mdx) to parse Stellar network transaction data and derive token transfer events. Before TTP, developers had to manually parse complex ledger data, operation results, and ledger entry changes to understand when and how assets moved between accounts, contracts, and other entities on the network.
+
+Prior to [CAP-67 Unified Events](https://stellar.org/protocol/cap-67), tracking token transfers required significant custom logic to handle different operation types, interpret ledger changes, and reconstruct the flow of assets. CAP-67 introduced a standardized event format that simplifies this process by providing a unified way to represent all token transfer activities.
+
+TTP serves as the implementation bridge for CAP-67, automatically generating these standardized events from Stellar ledger data. It can operate in two modes:
+
+- **Standalone mode**: TTP analyzes operations, operation results, and ledger entry changes to derive transfer events
+- **Unified events mode**: TTP reads directly from CAP-67 compliant unified events when available in the ledger data
+
+For more details on operational modes, see the [Modes of Operation](#modes-of-operation) section.
+
+## Key Features
+
+- Processes all token movement operations - classic and smart contract:
+
+  - Simple Payments
+  - Path payments
+  - Account merges
+  - Trustline Revocations
+  - Claimable balance operations
+  - Liquidity pool operations
+  - Clawback operations
+  - Smart contract events - Stellar Asset Contract events + SEP-41 compliant token events
```

Comment:
nit: link SEP-41 and expand on its definition, e.g. `[SEP-41, Token Interface](link)` like you did w/ CAP-67 (it's always good to provide a brief description/title when referencing a CAP/SEP for the first time)